frontend/src/metabase/nav/components/ReadOnlyBanner/ReadOnlyBanner.tsx

```diff
@@ -1,10 +1,17 @@
 import { t } from "ttag";
 
-import { Flex, Icon, Text } from "metabase/ui";
+import { Flex, Icon, Text, useMantineTheme } from "metabase/ui";
 
 export const ReadOnlyBanner = () => {
+  const theme = useMantineTheme();
   return (
-    <Flex py="0.75rem" px="1rem" bg="accent4" gap="md" align="center">
+    <Flex
+      py="0.75rem"
+      px="1rem"
+      bg={theme.fn.themeColor("accent4")}
```

Comment:
This change is to allow typing `CUSTOM_COLORS` in `frontend/src/metabase/ui/utils/colors.ts` to be easier since `accent4` and `accent5` is only used once in Mantine context.